packages/metamask-module-template/LICENSE

```diff
@@ -0,0 +1,20 @@
+MIT License
```

Comment:
Added using `create-package` script as an example package which we can remove in future